pubspec.yaml

```diff
@@ -1,6 +1,7 @@
 name: openfoodfacts
 description: Dart package for the Open Food Facts API, a food products database made by everyone, for everyone.
-version: 1.3.9
+# This is automatically taken care of, no need to increase
```

Comment:
can you add some more details? Where is it taken care of, if this version line is a no-op, is it possible to live without a version here?

Reply:
Updated the comment, is it better now